app/portainer/filters/filters.js

```diff
@@ -95,13 +98,13 @@ export function endpointTypeName(type) {
 
 export function environmentTypeIcon(type) {
   if (type === 3) {
-    return 'fab fa-microsoft';
+    return MicrosoftIcon;
```

Comment:
can you sync this function with the one on EE?

Reply:
updated

Reply:
this was reverted as well

Reply:
Sorry, I included this in the reverted commit because I thought you wanted to decrease the scope

Reply:
Updated bringing over the function from EE again